autoware_v2x_msgs/package.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>autoware_v2x_msgs</name>
+  <version>1.0.0</version>
+  <description>Autoware common messages package.</description>
+  <maintainer email="isamu.takagi@tier4.jp">Takagi, Isamu</maintainer>
```

Comment:
Could you add @yukkysaito  and @mitsudome-r  to the maintainers?

Reply:
@yukkysaito Add maintainters https://github.com/autowarefoundation/autoware_msgs/pull/77/commits/35755d7bb3b6ed2da71c32d0779e23f4e41ea4b0.